Source/CheckoutSdkIos.swift

```diff
@@ -8,13 +8,21 @@ func addKeyboardToolbarNavigation(textFields: [UITextField]) {
     // create the toolbar
     for (index, textField) in textFields.enumerated() {
         let toolbar = UIToolbar()
-        let prevButton = UIBarButtonItem(image: Constants.Bundle.Images.keyboardPrevious.image ?? UIImage(),
-                                         style: .plain, target: nil, action: nil)
+        let prevButton = UIBarButtonItem(
+            image: Constants.Bundle.Images.keyboardPrevious.image ?? UIImage(),
+            style: .plain,
+            target: nil,
+            action: nil)
```

Comment:
I would personally disable whatever rule caused this change as it may become very noisy. Its definitely well intended but I'd trust engineer to decide when needed to break down and align parameters and the level at which to do it.